canonical/ubuntu.com
# Redirect /engage/multi-cloud-guide to /engage/a-business-guide-to-cloud-migration

## Done

- Redirect /engage/multi-cloud-guide to /engage/a-business-guide-to-cloud-migration based on the [issue](https://warthogs.atlassian.net/browse/WD-5483)

## QA

- Go to https://ubuntu-com-13064.demos.haus/engage/multi-cloud-guide and see that it redirects https://ubuntu-com-13064.demos.haus/engage/a-business-guide-to-cloud-migration

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5483
